packages/blocks/src/api/registration.js

```diff
@@ -343,6 +357,24 @@ function translateBlockSettingUsingI18nSchema(
  * @param {Object} settings        The block collection settings.
  * @param {string} settings.title  The title to display in the block inserter.
  * @param {Object} [settings.icon] The icon to display in the block inserter.
+ *
+ * @example
+ * ```js
+ * import { __ } from '@wordpress/i18n';
+ * import { registerBlockCollection, registerBlockType } from '@wordpress/blocks';
+ *
+ * // Register the collection.
+ * registerBlockCollection( 'my-collection', {
```

Comment:
Do we have any detailed documentation for block collections? It would be great to have on presenting how to group blocks in the inserter by shared namespace.

Reply:
I don't believe we do. I have it on my list to create some.